### ISSUE TITLE
Teamfight icon on timeline not being shown on safari (and probably other browsers too)

### ISSUE THREAD

**Issue by dartharaknus** [odota/ui#601]
- **OS**: Mac
- **Screen Resolution**: 1080p
- **Browser/Version**: Safari
- **Link to page with the issue, or screenshot**: Every match

**Comment by mike-robertson** [comment]
I can't reproduce this in safari. Can you link a screenshot and your safari/mac version?

OS: yosemite 10.10.5
Safari: 9.0.1

![](http://i.imgur.com/NGsGmQS.jpg)

**Comment by dartharaknus** [comment]
OSX: 10.11.1
Safari: 9.1.2

And the teamfights page actually kills my browser almost. I suspect the "Measure"

**Comment by mike-robertson** [comment]
I did notice that performance is very terrible in safari. Not sure what's going on there, but it's an order of magnitude worse than chrome. I don't mind someone looking into it but I don't really think safari is a massive priority.

**Comment by mike-robertson** [comment]
Also can you include a screenshot?

**Comment by howardchung** [comment]
maybe we can do a quick test with disabled transitions and see if it improves safari perf?

**Comment by dartharaknus** [comment]
I can test it out if you have a branch for me.

**Comment by dartharaknus** [comment]
Here's the screenshot.

<img width="1207" alt="opendota_-_dota_2_statistics" src="https://cloud.githubusercontent.com/assets/5713044/21268428/0c523516-c3af-11e6-94bc-763a42f7db01.png">

**Comment by mike-robertson** [comment]
Cool, no console errors, right? And does this exist on the team fights page as well?

**Comment by dartharaknus** [comment]
Yup, no errors. And yup. Same on teamfights page. The teamfight icons on map don't show as well.

And I have to close the tab immediately otherwise, my browser becomes unresponsive
